### DISCUSSION TITLE
Set SearXNG Base URL Error 403

### DISCUSSION THREAD

**Discussion by Sakura4036** [langgenius/dify#4599] [Help] [answered]
Deploying Searxng with docker， on the same docker network as the dedify docker.

![image](https://github.com/langgenius/dify/assets/42060616/a64220b6-1740-4f40-b339-cfecc4323877)

**Comment by patryk20120** [comment]
Have you added `json` format as it states in docs? https://docs.dify.ai/tutorials/tool-configuration/searxng#id-2.-change-settings.yml

**Reply by Sakura4036** [reply]
yes, i did.  For permissions reasons, I went into the docker container and changed the `settings.yml`.

![image](https://github.com/langgenius/dify/assets/42060616/261ca3f0-b4bf-42c3-b093-9ddad500d2d1)

**Reply by patryk20120** [reply, marked as answer]
Have you restarted the container for changes to apply?

**Comment by jianhuatu** [comment]
官网文档有点问题，应该是这样
# remove format to deny access, use lower case.
# formats: [html, csv, json, rss]
search:
    formats:
      - html
      - json    # <-- add this line
